politeiad/api/v1/mime/bmime.go

```diff
@@ -10,11 +10,7 @@ import (
 var (
 	// validMimeTypesList is a list of all acceptable MIME types that
 	// can be communicated between client and server.
-	validMimeTypesList = []string{
-		"image/png",
-		"text/plain",
-		"text/plain; charset=utf-8",
-	}
+	validMimeTypesList = []string{}
```

Comment:
This list is redundant now that the valid MIME types have been moved to config.  The `validMimeTypesMap` should be populated directly from the config valid MIME types list.